release-notes.md

```diff
@@ -0,0 +1,55 @@
+# Release v0.7.1
+
+## New features
+* Add [alert manager](prometheus/doc/alert-manager.md), defined [several rules](https://github.com/Microsoft/pai/tree/pai-0.7.y/prometheus/prometheus-alert) for alert. Operator will receive email notification if cluster has some problem.
+
+## Improvements
+* [Optimize](https://github.com/Microsoft/pai/pull/1021) boot speed of web portal.
+* Do [not delete](https://github.com/Microsoft/pai/pull/1038) etcd data by default when upgrading kubernetes.
+* Documentation
+  * Restructured and refined README to provide a better experiences for new users.
+  * Documentation improvement for:
+    * [OpenPai deployment](https://github.com/Microsoft/pai/blob/pai-0.7.y/pai-management/doc/cluster-bootup.md)
+    * [REST Server](https://github.com/Microsoft/pai/blob/pai-0.7.y/rest-server/README.md)
+    * [Zookeeper](https://github.com/Microsoft/pai/blob/pai-0.7.y/zookeeper/zookeeper.md)
+    * [exporter](https://github.com/Microsoft/pai/blob/pai-0.7.y/prometheus/doc/README.md)
+    * [paictl design](https://github.com/Microsoft/pai/blob/pai-0.7.y/pai-management/doc/paictl-design.md)
+    * [kubernetes deployment Q&A](https://github.com/Microsoft/pai/blob/pai-0.7.y/pai-management/doc/kubernetes-deploy-qna.md)
+    * [pai-build](https://github.com/Microsoft/pai/blob/pai-0.7.y/pai-management/doc/pai-build.md)
+    * [Etcd maintenance](https://github.com/Microsoft/pai/blob/pai-0.7.y/pai-management/doc/etcd.md)
+    * [hdfs](https://github.com/Microsoft/pai/blob/pai-0.7.y/pai-management/doc/hdfs.md)
+* More examples:
+  * [caffe](https://github.com/Microsoft/pai/tree/pai-0.7.y/examples/caffe)
+  * [caffe2](https://github.com/Microsoft/pai/tree/pai-0.7.y/examples/caffe2)
+  * [chainer](https://github.com/Microsoft/pai/tree/pai-0.7.y/examples/chainer)
+  * [kafka](https://github.com/Microsoft/pai/tree/pai-0.7.y/examples/kafka)
+  * [Spark](https://github.com/Microsoft/pai/tree/pai-0.7.y/examples/spark)
+  * [XGBoost](https://github.com/Microsoft/pai/tree/pai-0.7.y/examples/XGBoost)
+
+## Bug fixes
+* Fix nginx reverse proxy [issue](https://github.com/Microsoft/pai/pull/1009) in webhdfs.
+* Fix pylon UI [issue](https://github.com/Microsoft/pai/issues/916).
+* Fix webportal data table [bug](https://github.com/Microsoft/pai/pull/734).
+
```

Comment:
I tuned the messages a little bit:

* Fixed nginx reverse proxy issue in webhdfs - [PR 1009](https://github.com/Microsoft/pai/pull/1009)
* Fixed pylon UI issue - [PR 916](https://github.com/Microsoft/pai/issues/916)
* Fixed webportal data table issue - [PR 734](https://github.com/Microsoft/pai/pull/734)